server/tests/miscellaneous/connection.spec.js

```diff
@@ -0,0 +1,38 @@
+import chai from "chai";
+import chaiHttp from "chai-http";
+import app from "../../index";
+
+chai.use(chaiHttp);
+const expect = chai.expect;
+
+describe("Server", () => {
+  it("should return Connection ok", done => {
```

Comment:
This is not descriptive enough. The test should tell me exactly what it is doing?

it("should return successful connection")